pontoon/base/__init__.py

```diff
@@ -1,13 +0,0 @@
-MOZILLA_REPOS = (
-    "ssh://hg.mozilla.org/users/m_owca.info/firefox-beta/",
-    "ssh://hg.mozilla.org/users/m_owca.info/firefox-for-android-beta/",
-    "ssh://hg.mozilla.org/users/m_owca.info/thunderbird-beta/",
-    "ssh://hg.mozilla.org/users/m_owca.info/lightning-beta/",
-    "ssh://hg.mozilla.org/users/m_owca.info/seamonkey-beta/",
-    "ssh://hg.mozilla.org/users/m_owca.info/firefox-central/",
-    "ssh://hg.mozilla.org/users/m_owca.info/firefox-for-android-central/",
-    "ssh://hg.mozilla.org/users/m_owca.info/thunderbird-central/",
-    "ssh://hg.mozilla.org/users/m_owca.info/lightning-central/",
-    "ssh://hg.mozilla.org/users/m_owca.info/seamonkey-central/",
-    "git@gitlab.com:seamonkey-project/seamonkey-central-l10n.git",
```

Comment:
Seamonkey is the only active project for which this special treatment still applies. I'm waiting for confirmation from them about whether to just drop this, or to add a `.l10n-ignore` file in the repo to get the same effect as currently.

Reply:
Agreed with Seamonkey to drop the special treatment.